### PR TITLE
Add Monktoberfest

### PR DESCRIPTION
**Pull request by pensarguet**
Add Monktoberfest

### CODE REVIEW

**Comment by micheelengronne** [comment]
You also changed devoxx uk. But this is the wrong domain. https://www.devoxx.co.uk is indeed the correct one.

File: README.md [hunk 1]
```diff
@@ -93,7 +94,7 @@ If they don't know your conference they can't buy a ticket and come to it 😏.
 
 ### November
 
-* 1-3: [Devoxx UK](https://www.devoxx.co.uk/) - London (UK)
+* 1-3: [Devoxx UK](https://www.devoxx.uk/) - London (UK)
```
Comment (micheelengronne) [inline README.md:97]: https://www.devoxx.co.uk is correct.

**Review by micheelengronne — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pensarguet) [reply README.md:97]: Ok, my bad! Sorry for that.

**Comment by pensarguet** [comment]
Correct the devoxx UK link to https://www.devoxx.co.uk/ following mail from Michée

File: README.md (hunk 1 above)
Comment (micheelengronne) [reply README.md]: No problem.

Comment (scraly) [reply README.md]: Yes devoxx.co.uk is the correct one ;-)

**Comment by scraly** [comment]
Thanks for your contribution :)

**Pull request merged**